src/routes/auth.route.js

```diff
@@ -0,0 +1,28 @@
+const authContoller = require('../controllers/auth.controller');
```

Comment:
There's a typo in the import statement: `authContoller` should be `authController` to match the correct spelling.